test/test_modules.py

```diff
@@ -304,6 +304,8 @@ def _traverse_obj(self, obj, func):
             return {name: self._traverse_obj(o, func) for name, o in obj.items()}
         elif isinstance(obj, (torch.Tensor, torch.nn.Parameter)):
             return func(obj)
+        elif isinstance(obj, (int, float, bool, complex, str)):
```

Comment:
Changed this because otherwise it was converting `Tuple[int]` to `Tuple[None]` in the noncontiguous outputs when converting lists of integers in `test_non_contiguous_tensors`

Reply:
good point